examples/cms-sanity/README.md

```diff
@@ -0,0 +1,140 @@
+# A statically generated blog example using Next.js and Sanity
+
+This example showcases Next.js's [Static Generation](/docs/basic-features/pages.md) feature using [Sanity](https://www.sanity.io/) as the data source.
+
+## Demo
+
+### [https://next-blog-sanity.now.sh/](https://next-blog-sanity.now.sh/)
+
+## How to use
+
+### Using `create-next-app`
+
+Execute [`create-next-app`](https://github.com/zeit/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
+
+```bash
+npm init next-app --example cms-Sanity cms-Sanity-app
+# or
+yarn create next-app --example cms-Sanity cms-Sanity-app
+```
+
+### Download manually
+
+Download the example:
+
+```bash
+curl https://codeload.github.com/zeit/next.js/tar.gz/canary | tar -xz --strip=2 next.js-canary/examples/cms-Sanity
+cd cms-sanity
+```
+
+## Configuration
+
+### Step 1. Create an account and a project on Sanity
+
+First, [create an account on Sanity](https://sanity.io).
+
+After creating an account, install the Sanity cli from npm `npm i -g @sanity/cli`.
+
+### Step 2. Create a new Sanity project
+
+In a separate folder run `sanity init` to initialize a new studio project.
+
+This will be where we manage our data.
+
+When going through the init phase make sure to select **Yes** to the **Use the default dataset configuration** step and select **Clean project with no predefined schemas** for the **Select project template** step.
+
+After initializing get the `projectId` and `dataset` values from the `sanity.json` file that was generated and add them to `lib/sanity.js` in the example.
```

Comment:
Maybe use `.env` instead? (If so, then we'll need to move "Step 5. Set up environment variables" earlier)

Reply:
Updated to use `.env` and moved the environment variables step up